<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lotnisko Rzeszów-Jasionka</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>1600</width>
     <height>1000</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>871</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkMonday">
         <property name="text">
          <string>Poniedziałek</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTuesday">
         <property name="text">
          <string>Wtorek</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkWednesday">
         <property name="text">
          <string>Środa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkThursday">
         <property name="text">
          <string>Czwartek</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkFriday">
         <property name="text">
          <string>Piątek</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSaturday">
         <property name="text">
          <string>Sobota</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSunday">
         <property name="text">
          <string>Niedziela</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFilter">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Filtruj loty</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputSearch">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Podaj dane lotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearch">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Szukaj lotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputArrival">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Podaj miejsce przylotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputDeparture">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Podaj miejsce wylotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputFlightCode">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Podaj numer rejsu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputTime">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Podaj godzinę lotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputFlightNumber">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Podaj numer lotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputCarrier">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Podaj przewoźnika</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputAircraftType">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Podaj typ samolotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddFlight">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Dodaj lot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeleteFlight">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Usuń lot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableFlights">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <column>
        <property name="text">
         <string>Numer lotu</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Wylot</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Przylot</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Godzina</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Przewoźnik</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nr rejsu</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Typ samolotu</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
